cmd/dist/fetch.go

```diff
@@ -113,12 +113,18 @@ func getResolver(ctx contextpkg.Context) (remotes.Resolver, error) {
 			prefix = strings.TrimPrefix(locator, "docker.io/")
 		)
 
+		if strings.HasPrefix(locator, "localhost:5000") {
```

Comment:
It doesn't need one.

Again, this is mostly throwaway. Let's not overanalyze so we can work towards our goal.